packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -65,9 +65,9 @@
   }
 }
 
-// Variants
```

Comment:
curious, why change the comments here? my understanding is that sass removes `//` comments (non standard CSS) at compile time which is nice since we don't need these for publish

Reply:
PostCSS can run into problems parsing SCSS-style and CSS single-line comments. Avoiding them entirely should prevent future headaches. 

For any pre-compiled builds (CSS only customers), the comments may be useful. For our React build, we could pull in https://github.com/sindresorhus/strip-css-comments or similar if the extra page weight is a problem. PostCSS may be able to handle this for us already.

Reply:
@edburyenegren-okta right I'm aware of the postcss plugins for comments and the concerns for our build. Parsing sass comments will not be a problem (i'm already doing it in my production build PR). Let's keep focused on the changes that offer the most value right now at this early stage.Building encapsulated component styles. Reducing the amount of "nice to have" refactors is critical at this point.

Reply:
These types of comments are already causing compilation errors. I prefer Sass comments because they're easier to write, but using them within CSS Modules that are cross-referenced causes a parsing error (import, value, extend, or include). I discovered this while refactoring and found that other people ran into the same issue.

As you note, using Sass is a temporary solution, which is an additional reason we should be using CSS-style comments.

Since you already have a solution for removing comments if desired, I'm unclear why this change is undesirable.

Reply:
Which package build is failing because of the double slash SASS comments?

Reply:
Storybook fails to build the components if one of the CSS modules uses Sass comments and is referenced by another module via a non-Sass function like `composes` or `@value`. AFAIK, this issue occurs any time `css-loader` is used in place of `scss-loader` when parsing a file, as others online have run into the same issue in other scenarios.

Even if we avoid those scenarios, is there a strong reason to preserve Sass comments as we move away from Sass?